chia/wallet/conditions.py

```diff
@@ -427,7 +427,7 @@ def __post_init__(self) -> None:
             and self.coin_id_committed is None
             and self.mode_integer is None
         ):
-            raise ValueError("Must specify at least one committment. Anyone-can-send/recieve is not allowed.")
+            raise ValueError("Must specify at least one commitment. Anyone-can-send/recieve is not allowed.")
```

Comment:
This change, while correct, requires changing some test code, as the test code looks for this exact matching string while testing this exception flow